ReadMe.md

```diff
@@ -28,13 +28,13 @@ Scenic View has releases for JDK 8, JDK 9, and JDK 11:
 
 For more information about JavaFX 11, see https://openjfx.io/openjfx-docs/.
 
-## Scenic View for JDK 11+
+## Scenic View for JDK 21+

```

Comment:
I'm confused - why are we increasing the baseline like this? What about people using earlier JDK versions?

Reply:
I can change it back to JDK 11. I'm assuming things will work successfully.

Reply:
Yeah - let's do the minimum necessary to support JDK 21, without breaking anything for people on earlier versions?